userspace/engine/falco_common.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <string>
+#include <exception>
+
+extern "C" {
+#include "lua.h"
+#include "lualib.h"
+#include "lauxlib.h"
+}
+
+#include <sinsp.h>
+
+//
+// Most falco_* classes can throw exceptions. Unless directly related
+// to low-level failures like inability to open file, etc, they will
+// be of this type.
+//
+
+struct falco_exception : std::exception
+{
+	falco_exception()
+	{
+	}
+
+	~falco_exception() throw()
```

Comment:
I was just following the example of sinsp_exception. Should I change both to be virtual and to drop the exception specification?